src/Poseidon/CLI/Fetch.hs

```diff
@@ -187,19 +188,19 @@ printDownloadProgress fileSizeMB = loop 0 0
             x <- await
             maybe (return ()) (showDownloaded fileSizeMB loadedB) x
             where
-                showDownloaded fileSizeMB loadedKB x = do
-                    let newLoadedB = loadedB + B.length x
+                showDownloaded fileSizeMB_ loadedB_ x = do
```

Comment:
Here and in the following: Please check @nevrome whether this is all correct.

Reply:
Good spotting! Not sure why that worked, but your solution works as well, so I will just accept that :shrug:.